exp_build/plugin-config.testing.js

```diff
@@ -39,7 +35,30 @@ const config = [
     repo: "https://github.com/virtual-labs/tool-validation",
     tag: "v1.0.1",
     command: "npm i && node js/link_validation.js",
-  }
+  },
+  {
+    id: "svc-rating",
+    scope: PluginScope.PAGE,
+    repo: "https://github.com/virtual-labs/svc-rating",
+    tag: "v1.0.4",
+    label: "Validation Tool",
+    js_modules: [
+      "./index.js",
+      "./config.js",
+      "https://apis.google.com/js/api.js",
+    ],
+    css_modules: [
+      "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css",
+    ],
+    attributes: {
+      spreadsheetID: "1x12nhpp0QvnsA6x-O1sV4IA9SAbfVsq_wiexWkutOmU",
```

Comment:
Why do we need spreadsheetID here? Isn't it passed as argument to the component

Reply:
The handlebar for rating display component takes the attributes specified in the config file. The rating display handlebar then gets rendered in the html pages.